### PR TITLE
[MO] Pip installation message for not satisfied dependencies

### PR DESCRIPTION
**Pull request by yekruglov**
Description: 
MO suggests to use `install_prerequisites_[FRAMEWORK]` script for not satisfied versions of dependencies. It looks like 
`Please install required versions of components or use install_prerequisites script`
`<PATH_TO_OV>/openvino/tools/mo/install_prerequisites/install_prerequisites_caffe.sh`. Instead of that MO should advise to use pip installation with `extra` option when it is possible. More information [here](https://pypi.org/project/openvino-dev/).
The message will be look like 
`Please install required versions of components or run pip installation`
`pip install openvino-dev[caffe]`

Ticket: CVS-76520

Code:
* [x]  Comments
* [x]  Code style (PEP8)
* [x]  Transformation generates reshape-able IR N/A
* [x]  Transformation preserves original framework node names N/A
* [x]  Transformation preserves tensor names N/A

Validation:
* [x]  Unit tests N/A
* [x]  Framework operation tests N/A
* [x]  Transformation tests N/A
* [x]  Model Optimizer IR Reader check N/A

Documentation:
* [x]  Supported frameworks operations list N/A
* [x]  Guide on how to convert the **public** model N/A
* [x]  User guide update N/A

### CODE REVIEW

File: tools/mo/openvino/tools/mo/utils/versions_checker.py [hunk 1]
```diff
@@ -287,9 +285,17 @@ def check_requirements(framework=None):
             continue
 
     if len(not_satisfied_versions) != 0:
-        extension = 'bat' if os.name == 'nt' else 'sh'
-        install_file = 'install_prerequisites{0}.{1}'.format(framework_suffix, extension)
-        helper_command = os.path.join(os.path.dirname(requirements_file), 'install_prerequisites', install_file)
+
+        if framework == 'tf':
```
Comment (slyubimt) [inline tools/mo/openvino/tools/mo/utils/versions_checker.py:289]: Why Caffe and other supported frameworks are not covered?
https://github.com/openvinotoolkit/openvino/blob/bc70b2b68b849503815466a30a14333a2b32b12b/tools/mo/openvino/tools/mo/main.py#L236

File: tools/mo/openvino/tools/mo/utils/versions_checker.py [hunk 2]
```diff
@@ -287,9 +285,17 @@ def check_requirements(framework=None):
             continue
 
     if len(not_satisfied_versions) != 0:
-        extension = 'bat' if os.name == 'nt' else 'sh'
-        install_file = 'install_prerequisites{0}.{1}'.format(framework_suffix, extension)
```
Comment (mvafin) [inline tools/mo/openvino/tools/mo/utils/versions_checker.py:291]: We removing hint to use install_prerequisites if some dependencies aren't satisfied, instead we ask to use pip install openvino-dev, but installation through pip is not the only way to install the openvino, are we sure this is correct to ask to use pip?

Comment (popovaan) [reply tools/mo/openvino/tools/mo/utils/versions_checker.py:294]: Currently we have only pip. Other sources of distributions like yum, apt, pset were removed. Regular installer installs pip packages as well.

Comment (popovaan) [reply tools/mo/openvino/tools/mo/utils/versions_checker.py:294]: I have clarified this with @slyubimt

**Review by popovaan — APPROVED** [review]

**Review by sadolini — APPROVED** [review]

File: tools/mo/openvino/tools/mo/utils/versions_checker.py [hunk 3]
```diff
@@ -290,9 +288,17 @@ def check_requirements(framework=None):
...
-        helper_command = os.path.join(os.path.dirname(requirements_file), 'install_prerequisites', install_file)
+
+        if framework == 'tf':
+            extra = '[tensorflow]'
+        elif framework == 'tf2':
+            extra = '[tensorflow2]'
+        elif framework is None:
+            extra = ''
+        else:
+            extra = '[{}]'.format(framework)
```
Comment (rkazants) [inline tools/mo/openvino/tools/mo/utils/versions_checker.py:299]: Please add extra for other frameworks: caffe, kaldi, mxnet, onnx

**Review by rkazants — APPROVED** [review]

**Pull request merged**